Tests/SocketTests/SocketTests.swift

```diff
@@ -1125,6 +1125,18 @@ class SocketTests: XCTestCase {
 		}
 	}
 	
+	func testFDSetBitFields() {
```

Comment:
Does this need adding to the `allTests` array too?

Reply:
Oops, right you are. That's done now.